src/wp-admin/admin-header.php

```diff
@@ -170,8 +170,8 @@
 	$admin_body_class .= ' taxonomy-' . $current_screen->taxonomy;
 }
 
-$admin_body_class .= ' branch-' . str_replace( array( '.', ',' ), '-', (float) get_bloginfo( 'version' ) );
-$admin_body_class .= ' version-' . str_replace( '.', '-', preg_replace( '/^([.0-9]+).*/', '$1', get_bloginfo( 'version' ) ) );
+$admin_body_class .= ' branch-' . str_replace( array( '.', ',' ), '-', (string) ( (float) get_bloginfo( 'version' ) ) );
+$admin_body_class .= ' version-' . str_replace( '.', '-', (string) preg_replace( '/^([.0-9]+).*/', '$1', get_bloginfo( 'version' ) ) );
```

Comment:
`preg_replace()` will always return a string when given a string, except when the regex errored out and this regex does not look like on which will error.